<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvironmentProbeProperty</class>
 <widget class="QWidget" name="EnvironmentProbeProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox
{
	background-color:transparent;
    border-bottom-left-radius: 3px;
    border-bottom-right-radius: 3px;
    border: 1px solid #005192;
    border-top: none;
    margin-top: 18px;
}
 
QGroupBox::title
{
	subcontrol-origin: margin;	
    background:url(:/assets/subWindowBackground);
    border-top-left-radius: 3px;
    border-top-right-radius: 3px;
    border: 1px solid #005192;
    border-bottom: none;
   background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #005192, stop: 1 #002f60);
    padding: 2 5000 2 2; /* some large value, as QGrouBox actual width is not known */
    color: white; 
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="ScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>255</width>
        <height>470</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="0" column="0">
        <widget class="GroupBox" name="Enable">
         <property name="title">
          <string>Enable</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <property name="spacing">
           <number>3</number>
          </property>
          <item row="0" column="0">
           <widget class="GroupBox" name="groupBox">
            <property name="title">
             <string>Output</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <property name="spacing">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Path:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Map Size:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="MapSize">
               <item>
                <property name="text">
                 <string>1 x 1</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2 x 2</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4 x 4</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8 x 8</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>16 x 16</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>32 x 32</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>64 x 64</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>128 x 128</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>256 x 256</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>512 x 512</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1024 x 1024</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2048 x 2048</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4096 x 4096</string>
                </property>
                <property name="icon">
                 <iconset resource="geditor.qrc">
                  <normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</normaloff>:/Resources/MetroStyle/appbar.arrow.expand.png</iconset>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QPushButton" name="Bake">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Bake</string>
               </property>
               <property name="icon">
                <iconset resource="geditor.qrc">
                 <normaloff>:/Resources/MetroStyle/cooking_pot.png</normaloff>:/Resources/MetroStyle/cooking_pot.png</iconset>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>1</number>
               </property>
               <item>
                <widget class="QLineEdit" name="OutputPath">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="SelectOutputPath">
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="GroupBox" name="groupBox_3">
            <property name="title">
             <string>Bound</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <property name="spacing">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Radius:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="DoubleSpinBox" name="Radius">
               <property name="decimals">
                <number>2</number>
               </property>
               <property name="maximum">
                <double>999999999.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.010000000000000</double>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="ColorSelector" name="BoundColor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QCheckBox" name="BoundVisible">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <property name="text">
                <string>Visible</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="GroupBox" name="groupBox_2">
            <property name="title">
             <string>Map</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <property name="spacing">
              <number>3</number>
             </property>
             <item row="0" column="0">
              <widget class="PictureBox" name="Map">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>128</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>128</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="acceptDrops">
                <bool>true</bool>
               </property>
               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="filter">
                <string>Texture files(*.dds)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureBox</class>
   <extends>QLabel</extends>
   <header>PictureBox.h</header>
  </customwidget>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>DoubleSpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>ScrollArea</class>
   <extends>QScrollArea</extends>
   <header>ScrollArea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorPreview</class>
   <extends>QWidget</extends>
   <header>ColorPreview.h</header>
  </customwidget>
  <customwidget>
   <class>ColorSelector</class>
   <extends>ColorPreview</extends>
   <header>ColorSelector.h</header>
  </customwidget>
  <customwidget>
   <class>GroupBox</class>
   <extends>QGroupBox</extends>
   <header>GroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="geditor.qrc"/>
 </resources>
 <connections/>
</ui>
